scrapers_next/ma/committees.py

```diff
@@ -0,0 +1,100 @@
+from spatula import URL, HtmlListPage, HtmlPage, CSS, XPath, selectors
+from openstates.models import ScrapeCommittee
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        try:
```

Comment:
If possible, let's keep `for` loops out of `try` blocks. It isn't critical, and if this is the right solution, that's fine. But having a huge `try` block usually makes it harder to debug errors later.